mep/people/models.py

```diff
@@ -324,6 +324,13 @@ def viaf(self):
         if self.viaf_id:
             return ViafEntity(self.viaf_id)
 
+    @property
+    def short_name(self):
+        '''Shortened form of name used for locations where space is tight,
```

Comment:
Probably more useful to describe *what* this is (i.e. last name if we can determine it based on comma) than where it is currently used.